.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### Type of change
+
+_Select the type of your PR_
+
+- [ ] Bugfix
+- [ ] Enhancement / new feature
+
+
+### Description
+
+_Please describe your pull request_
+
+### Checklist
+
+_Please go through this checklist and make sure all aplicable tasks have been done_
```

Comment:
typo: applicable

Also on the Conversation tab on this PR the leading `_` is being rendered. Not sure why.

Reply:
_Applicable_ fixed. The `_` was typo which I fixed already before - I was missing the closing end.